### ISSUE TITLE
Install f90wrap without pip because there is no network in the cluster and it is under a personal account? 

### ISSUE THREAD

**Issue by ctxu-NWPU** [jameskermode/f90wrap#224]
Can i install f90wrap without pip because there is no network in the cluster and it is under a personal account? I need to install QUIP and run LAMMPS for molecular dynamics simulation. @jameskermode

**Comment by jameskermode** [comment]
Yes, just download either the source tar ball or the relevant .whl file from PyPI and copy it over to the machine then install with pip locally.

**Comment by jameskermode** [comment]
To build from source you need pip and meson. Note that you need to use an older release for compatibily with QUIP: v0.2.14 or earlier. You could download a wheel from https://github.com/jameskermode/f90wrap/releases/tag/v0.2.14, but you will need pip install locally to be able to install it.
